docs/config/roddyConfigurationOptions.rst

```diff
@@ -0,0 +1,62 @@
+Roddy Configuration Options
+===========================
+
+The following variables are used by Roddy itself:
+
+Path Configuration
+------------------
+
+The following variables describe paths and may contain metadata-variable references such as '${dataset}' that are filled in during the identification
+of input data during start up or filled with metadata values during submission.
+
+* inputBaseDirectory:  This option is also set by the --useiodir=$outputBaseDirectory,$inputBaseDirectory command-line option.
+* outputBaseDirectory: This option is also set by the --useiodir=$outputBaseDirectory[,$inputBaseDirectory] command-line option. Usually, this directory contains
+    * one subdirectory per dataset (e.g. patient).
+    * the "execution cache file" (.roddyExecCache.txt) which containing the list of executed runs
+    * the global "execution store" (.roddyExecutionStore) with the decompressed plugins used for the analyses.
```

Comment:
It's not the plugins but their tool folders. This is done for storage reasons. Maybe we should copy the whole plugin in the future.